### ISSUE TITLE
iOS App Store version

### ISSUE THREAD

**Issue by Iamthebestintheworld222** [jpd002/Play-#1371]
Could you add App Store support for the emulator since apple now supports emulator apps?

**Comment by jpd002** [comment]
That's something I would like doing, but I think JIT is still not offically allowed on iOS... so, until then, not something that will happen.

**Comment by SternXD** [comment]
Yeah, at the moment, the get-task-allow entitlement is not allowed for App Store applications.

**Comment by Samuel525432** [comment]
Maybe you can use Interpreter or game mode or something to make the emu come to the App Store
